### ISSUE TITLE
Add subtitles to MP4 file

### ISSUE THREAD

**Issue by svnpenn** [axiomatic-systems/Bento4#248]
Mp4Box can do this:

~~~
mp4box -add xr.mp4 -add xr.en.srt -new ya.mp4
~~~

I did not see a Bento4 tool that can do this

**Comment by barbibulle** [comment]
Good point. Bento4 has some support for subtitles, but not yet the ability to "import" subtitles files from non-mp4 formats, like 'srt'. 
I'll mark that as a feature to add, it should be fairly simple to do.
